src/transaction/log_applier.c

```diff
@@ -7082,13 +7101,14 @@ la_log_page_check (const char *database_name, const char *log_path, INT64 page_n
       if ((res <= 0) || (ha_apply_info.creation_time.date == 0 && ha_apply_info.creation_time.time == 0))
 	{
 	  error = res;
-
+	  *check_applied_info = false;
+	  printf ("\n *** Applied Info. *** \n");
+	  er_set_with_oserror (ER_ERROR_SEVERITY, ARG_FILE_LINE, ER_INTERFACE_NO_AVAILABLE_INFORMATION, 0);
```

Comment:
What kind of os error do you expect to be printed here?

Reply:
Why do you choose `ER_INTERFACE_NO_AVAILABLE_INFORMATION` which is used in ci or api module?

Reply:
@wrlawodms I consider and consider again and then ER_INTERFACE_NO_AVAILABLE_INFORMATION is the best error message :(